content/english/web-basics/activity-4-javascript.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Activity 4 - JavaScript"
+date: 2024-09-16T15:25:51.25-07:00
+draft: true
+weight: 9
+---
+<!-- <p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/tppIJbxknJc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p> -->
+
+## Making your webpage interactive with JS
+
+Now that we have elements on our screen and know how to style them with CSS, let's make our website more interesting and interactive. We may want to add animations or buttons that our site visitors can click on. We can do this by adding code logic to our page using JavaScript.  
+
+## JavaScript (JS)
+
+JavaScript is a programming language often used to make websites interactive and control their behavior. It allows you to add logic to your page, for example, if you click a button, then something on the page changes in response!
+
+<img src="../media/clickingButton.png" alt="image of clicking a button" width="200" height="200">
+
+<!-- ![alt text](../media/clickingButton.png "image of clicking a button") -->
+
+We'll cover the very basics to get your started.
+
+Similar to CSS, JS files work closely with HTML tags to select different items on the page.
+
+See this example below:
+<p class="codepen" data-height="300" data-default-tab="js,result" data-slug-hash="PorrJbr" data-pen-title="Adding JavaScript" data-preview="true" data-user="houghj16" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+  <span>See the Pen <a href="https://codepen.io/houghj16/pen/PorrJbr">
+  Adding JavaScript</a> by Jessie Houghton (<a href="https://codepen.io/houghj16">@houghj16</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
+
+<b>Click the "HTML" tab on the top left corner to see the HTML code, the "CSS" tab to see the CSS code, and the "JS" tab to see the JS code.</b>
+
+Remember how we used the `class` attribute or  `id` attribute with CSS to give the corresponding HTML tags a style? We can use that same id with JavaScript to select that element.
+
+You'll notice in the example above, even though we made the info div teal in exercise 2, since the JavaScript function <code>document.getElementById("info").style.color = "black";</code> runs after the styling is applied, it changed the color back to black.
```

Comment:
```suggestion
You'll notice in the example above, even though we made the info `div` teal in exercise 2, since the JavaScript function <code>document.getElementById("info").style.color = "black";</code> runs after the styling is applied, it changed the color back to black.
```

Reply:
I'd see if we can emphasize that this is another way to change the style of an element, although CSS is preferred.